src/jsd/actuator_fsm_helpers.cc

```diff
@@ -535,10 +535,14 @@ fastcat::FaultType fastcat::Actuator::ProcessFaulted()
 
 fastcat::FaultType fastcat::Actuator::ProcessHalted()
 {
-  if (IsIdleFaultConditionMet()) {
-    ERROR("Act %s: %s", name_.c_str(), "Fault Condition present, faulting");
-    return ALL_DEVICE_FAULT;
-  }
+  //  Invoking a RESET will fail the following fault condition since the 
```

Comment:
I'm not following this comment. It seems to me the condition below will fail independently of invoking a reset if there is a latched error. Also, `EgdReset()` is no longer used in the code.

Reply:
`EgdReset() is handled here: https://github.com/nasa-jpl/fastcat/blob/81de32c9d635e2af37e1afc37fa5fca5a437c326/src/jsd/actuator_fsm_helpers.cc#L30

Reply:
I agree the comments can be revised with the associated changes to `jsd`

Reply:
I just realized something. If we already call `EgdClearErrors` in `EgdReset`, then we wouldn't need to skip `IsIdleFaultConditionMet` here. The only relevant transition into `ACTUATOR_SMS_HALTED` comes from `Actuator::Reset`. The other two transitions in that state, are when exiting `ACTUATOR_SMS_HOLDING` and the handler of Halt commands, both of which unable to get to that transition if the drive was faulted (i.e. no errors would be latched).

Reply:
great point, I'll test it out today.

Reply:
I re-enabled all fault checks, good eye on this!